canonical-web-and-design/maas-ui
# fix(ui): scale the doughnut chart with the font size

## Done

- Update the doughnut chart to fit within its container.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Navigate to a lxd pod details page that shows a doughnut chart.
- Change your browser's font size to something smaller than the default.
- The doughnut chart should resize and not overlap the content next to it.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1588
